pdk/1.0/modules/ROOT/pages/policies-pdk-apply-policies.adoc

```diff
@@ -12,19 +12,74 @@ To learn more about defining configuration parameters, see xref:policies-pdk-cre
 
 == Before You Begin
 
-* xref:policies-pdk-publish-policies.adoc[Upload your Custom Policy to Exchange]
+* xref:policies-pdk-compile-policies.adoc[Compile your custom policy].
+* If applying your policy to Flex Gateway running in Connected Mode, xref:policies-pdk-publish-policies.adoc[upload your custom policy to exchange].
 
-== Connected Mode
+== Apply Policies in Connected Mode
 
 When you apply a custom policy in Connected Mode, the configuration parameters and metadata defined in `gcl.yaml` appear in the API Manager UI.
 
 To apply a custom policy in Connected Mode, see xref:gateway::policies-included-apply.adoc[Apply a Policy in Connected Mode]. 
 
-== Local Mode
+=== Update Policies in Connected Mode
+
+To update a custom policy in API Manager, you can either:
+
+* Delete the policy, and then re-apply the latest version of the policy. This automatically updates the policies binary file.
+* Update the version of the applied policy. If you update the version on the applied policy, you must check for implementation updates to ensure your policy's binary file is up to date.
++
+To check for implementation updates, see xref:exchange::manage-versions.adoc#update-policy-implementations[Update Policy Implementations].
+
+[[apply-policies-in-local-mode]]
+== Apply Policies in Local Mode
 
 For Flex Gateway running in Local Mode, you apply custom policies using xref:gateway::flex-local-configuration-reference-guide.adoc[local declarative configuration files]. You specify the location of these configuration files when you first run Flex Gateway.
 
-To apply a custom policy in Local Mode, use either xref:gateway::flex-local-secure-api-with-basic-auth-policy.adoc[] or xref:gateway::flex-local-secure-api-with-auto-policy.adoc[] as an example. Replace the example configuration with your policy's configuration parameters.
+To create the local declarative files:
+
+. In the `target/wasm32-wasi/release` PDK directory, copy both of the following files:
++
+* `<your_custom_policy>_definition.yaml`
+* `<your_custom_policy>_implementation.yaml`
++
+. Paste the two files in the `/etc/mulesoft/flex-gateway/conf.d/` Flex Gateway configuration directory.
++
+. Create a policy binding configuration file with a `.yaml` file extension to bind the policy to an API instance:
++
+* Give the file a custom name.
+* Save the file in the Flex Gateway configuration directory `/etc/mulesoft/flex-gateway/conf.d/custom`. This directory can contain multiple configuration files.
+. Copy and paste the following YAML snippet into the file, substituting your values where indicated:
++
+[source,yaml]
+----
+apiVersion: gateway.mulesoft.com/v1alpha1
+kind: PolicyBinding
+metadata:
+  name: <custom-policy-id>
+spec:
+  targetRef:
+    kind: ApiInstance
+    name: <your-api-instance>
+  policyRef:
+    kind: Extension
+    name: <your_custom_policy>-implementation
```

Comment:
This name won't be like this. It will be more like `<your_custom_policy>-v1-0-impl`